server/proxies/api.js

```diff
@@ -9,9 +19,19 @@ module.exports = function(app) {
     console.error(err, req.url); // eslint-disable-line no-console
   });
 
+  proxy.on('proxyRes', async function (proxyRes, req, res) {
+    var gunzip = zlib.createGunzip();
+    let body = await streamToString(proxyRes.pipe(gunzip));
+
+    return res.end(body.replace(/https:\/\/localhost:4200/g, 'http://localhost:4200'));
```

Comment:
No particular reason 🤔 I guess I just included it so that it wouldn't be over zealous in its replacements. 

We could just get it to replace `https://localhost:` with `http://localhost:` (ignoring any ports) if you think that would be safe enough?

Reply:
I think it should be -- `https://localhost` is rare enough, and it is only development. But `:4200` would probably only work for a few people running in dev.

Reply:
👍 I have updated it now